decode_test.go

```diff
@@ -1,4 +1,5 @@
 //
+
```

Comment:
Please revert this change, or explain why you did it

Reply:
Ah, this is my bad. I'll force push the next change reverting this.